src/pages/Sign Up/signup.jsx

```diff
@@ -0,0 +1,92 @@
+import "../Nav/nav.css";
+import "./signUp.css";
+import {IcBaselineRemoveRedEye} from "../../assets/icons/icons.jsx";
+import { useState } from "react";
+import {Link} from "react-router-dom";
+
+import axios from "axios";
+import { useAuth } from "../../context/Authentication";
+import { useReducer } from "react/cjs/react.development";
+
+export function SignUp(){
+   const[passwordType,setPasswordType]=useState("password");
+   const[state,dispatch]=useReducer(details,{firstName:"",lastName:"",email:"",password:"",confirmPassword:""});
```

Comment:
you can use detailReducer instead of `details` for good naming structure

Reply:
thanks for suggestion